src/main/windows.ts

```diff
@@ -166,28 +186,39 @@ export const openAppWindow = (app: LaunchableApp) => {
     });
 };
 
+export const openApp = (app: AppSpec, openAppOptions?: OpenAppOptions) => {
+    if (app.source === LOCAL) {
+        openLocalAppWindow(app.name, openAppOptions);
+    } else {
+        openDownloadableAppWindow(app, openAppOptions);
+    }
```

Comment:
```suggestion
    if (app.source === LOCAL) {
        openLocalAppWindow(app.name, openAppOptions);
        return;
    }
    
    openDownloadableAppWindow(app, openAppOptions);
```

Reduce nesting by this technique

Reply:
Does it actually reduce _nesting_? The code is still nested to the same level, but you've just reduced the last else block, which doesn't really add or remove anything in my opinion. For the sake of the argument, take for example this ridiculous code:

```javascript
if (app.source === LOCAL) {
        openLocalAppWindow(app.name, openAppOptions);
        if (app.source !== LOCAL) {
                openDownloadableAppWindow(app, openAppOptions);
        }
}
```

Had the code looked like the one above, then you would have reduced the nesting. But including or excluding an else statement at the end, in this situation, is more about preference, in my opinion.